base/path.jl

```diff
@@ -86,7 +91,7 @@ julia> isabspath("home")
 false
 ```
 """
-isabspath(path::String) = ismatch(path_absolute_re, path)
+isabspath(path::AbstractString)
```

Comment:
Unless I made a typo, this is the signature embedded in the doc-string itself. I'm not entirely certain why (or if) it needs to be duplicated, but this function itself will be defined later in the file in an `@eval` loop.